nuitka/freezer/Standalone.py

```diff
@@ -669,7 +669,7 @@ def _detectBinaryPathDLLsPosix(dll_filename):
     return sub_result
 
 
-def _detectBinaryPathDLLsMacOS(original_dir, binary_filename):
+def _detectBinaryPathDLLsMacOS(original_dir, binary_filename, keep_unresolved=False):
```

Comment:
This keep_unresolved is not used, right? Probably has no place as we expect everything to resolve?

Reply:
No, is it used to get the original `@rpath` for `fixupBinaryDLLPaths()`. If it is passed the resolved paths, then `install_name_tool` won't be able to make the changes.